tests/unit/query_model/test_population_validation.py

```diff
@@ -157,7 +157,7 @@ def test_rejects_basic_unreasonable_population():
     ),
     (
         True,
-        Function.In(Value(10), Value({30, 20, 10})),
+        Function.In(Value(10), Value(frozenset([30, 20, 10]))),
```

Comment:
Can we validate this in a constructor?

Reply:
I was going to say that that's fiddly to do and low value, but then I realised it's really easy! Will add a commit.

Reply:
Commit here:
https://github.com/opensafely-core/databuilder/pull/1090/commits/d061c7e3b7e6263848ce3a11ba90a2b5add9f4e6